### DISCUSSION TITLE
I'm stuck. :/

### DISCUSSION THREAD

**Discussion by Chris-Xavier** [community/community#147017] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello Everyone,

So basically, for the past month or so, I have not been able to actually program. 
I'm in this CTE program for computer programming, right? But I haven't actually been able to like, sit down and program.
I've wanted to start building a social media application for a while now. But each time I try to sit down and just program anything,
I can't. It's like I don't remember anything and I'm starting to question if programming is right for me...?

Anyways, I wanted to get some thoughts on this and maybe it'll help me figure out where to start and how. 
Any ideas would be appreciated! Thank you!

**Comment by AguHDS** [comment]
i don't know your age but if you're young is not bad to prove things to see if it's for you. If you like programming, you'll spend entire days programming and getting better, if you don't, then you're going to know that, just try to not get frustrated and give it a try
